.cspell.json

```diff
@@ -126,7 +126,8 @@
     "aggr",
     "tcsetattr",
     "pysqlite",
-    "AADSTS700082"
+    "AADSTS700082",
+    "levelno"
```

Comment:
Note we should add telemetry notic:
○ Telemetry Notice. If the release has code that enables Microsoft to collect telemetry, also include in the repository:  [Task 2573599](https://msdata.visualstudio.com/Vienna/_workitems/edit/2573599): Telemetry Notice
			§ instructions to turn off the telemetry;
this [telemetry/data collection notice](https://docs.opensource.microsoft.com/releasing/general-guidance/telemetry) in a the README.md file.

Reply:
one example: https://github.com/microsoft/onefuzz/blob/58da7b431e0b307155b508eeee5922f23b209c46/README.md?plain=1#L79

Reply:
Updated and referenced `azure-cli`'s behavior; 
![image](https://github.com/microsoft/promptflow/assets/7776147/0a53f320-1231-40ea-b877-42bbb0793bfd)